rewrite-java/src/main/java/org/openrewrite/java/cleanup/AddInputStreamBulkReadMethod.java

```diff
@@ -0,0 +1,499 @@
+/*
+ * Copyright 2025 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.cleanup;
```

Comment:
This package only contains a few visitors at this moment, as we've moved the majority of the recipes that were formerly here into rewrite-static-analysis.

For this recipe I'm doubting on where to place it, but I feel perhaps https://github.com/openrewrite/rewrite-migrate-java/blob/b566417e5ef76dfa6b4026cb2c08b9a5b335cbe7/src/main/java/org/openrewrite/java/migrate/io is more fitting than having it here.

If we keep it here, it would also be included with any other language module as those tend to use `rewrite-java`, which feels a bit off.